applications/aks/default/gitops/values/backend-services/chat.yaml

```
@@ -23,6 +23,7 @@ cronJob:
 env:
   AMQP_BENCHMARK_PREFETCH: 1
   ANONYMIZED_CHAT_INTERACTIONS_EXPORT: "false"
+  APP_REPOSITORY_URL: http://node-app-repository.unique.svc.cluster.local:8088
```

This needs to change - the default port is used if nothing is specified so the app-repo internal address is not running on 8088.

```suggestion
  APP_REPOSITORY_URL: http://node-app-repository.unique.svc
```

---

nice catch!